Package.swift

```diff
@@ -3,6 +3,6 @@ import PackageDescription
 let package = Package(
   name: "Commandant",
   dependencies: [
-    .Package(url: "https://github.com/antitypical/Result.git", "2.0.0")
+    .Package(url: "https://github.com/antitypical/Result.git", "3.0.0-alpha.4")
```

Comment:
Unfortunately, SPM could not fetch from specific branch.
So it's using different Result version between Xcode and SPM.